spec/cypress_on_rails/vcr/insert_eject_middleware_spec.rb

```
@@ -0,0 +1,177 @@
+      end
+
+      describe '/__e2e__/vcr/eject' do
+        before do
+          env['PATH_INFO'] = '/__e2e__/vcr/eject'
+        end
+
+        it do
+          aggregate_failures do
+            expect(response).to eq([201,
+                                    { 'Content-Type' => 'application/json' },
+                                    ['{"message":"OK"}']])
```

rubocop

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)